test/OrchardCore.Tests/Apis/Lucene/LuceneQueryTests.cs

```diff
@@ -0,0 +1,118 @@
+
+                string titleBoostQuery = baseQuery.Replace("{0}", titleBoost).Replace("{1}", bodyBoost);
+                var contentItems = await SearchItems(scope, titleBoostQuery, "ArticleIndex");
+
+                Assert.True(contentItems.All(x => x.As<TitlePart>().Title.Contains("Orchard", StringComparison.OrdinalIgnoreCase)));
+
+                // Articles with "Orchard" in Title should NOT appear first
+                titleBoost = string.Empty;
+                bodyBoost = "^2";
+
+                string bodyBoostQuery = baseQuery.Replace("{0}", titleBoost).Replace("{1}", bodyBoost);
+                contentItems = await SearchItems(scope, bodyBoostQuery, "ArticleIndex");
+
+                Assert.False(contentItems.All(x => x.As<TitlePart>().Title.Contains("Orchard", StringComparison.OrdinalIgnoreCase)));
+            });
+        }
+
+        [Fact]
+        public async Task SimpleQueryWildcardHasResults()
+        {
+            await SiteContext.ExecuteTest<LuceneContext>(async scope =>
+            {
+                // Act
+                // Should find articles with "Orchard" in the title
+                string query = "{ \"from\": 0, \"size\": 1, \"query\": { \"simple_query_string\": { \"analyze_wildcard\": true, \"fields\": [ \"TitlePart\" ], \"query\": \"orch*\" } }}";
+
+                var contentItems = await SearchItems(scope, query, "ArticleIndex");
+
+                Assert.NotEmpty(contentItems);
+                Assert.Contains("Orchard", contentItems.First().As<TitlePart>().Title, StringComparison.OrdinalIgnoreCase);
+            });
+        }
+
+        [Fact]
+        public async Task WildcardQueryWithBoostHasResults()
+        {
+            await SiteContext.ExecuteTest<LuceneContext>(async scope =>
+            {
+                // Act
+                // Should find articles with "Orchard" in the title
+                string query = "{ \"from\": 0, \"size\": 10, \"query\": { \"bool\": { \"should\": [ { \"wildcard\": {  \"TitlePart\": { \"value\": \"orch*\", \"boost\": 2 } } }, { \"wildcard\": { \"HtmlBodyPart\": { \"value\": \"orchar*\", \"boost\": 5 } } } ] } } }";
+
+                var contentItems = await SearchItems(scope, query, "ArticleIndex");
+
+                Assert.NotEmpty(contentItems);
+                Assert.True(contentItems.Count() >= 4);
+
+                Assert.Contains("Orchard", contentItems.ElementAt(0).As<HtmlBodyPart>().Html, StringComparison.OrdinalIgnoreCase);
+                Assert.Contains("Orchard", contentItems.ElementAt(1).As<HtmlBodyPart>().Html, StringComparison.OrdinalIgnoreCase);
+                Assert.Contains("Orchard", contentItems.ElementAt(contentItems.Count() - 2).As<TitlePart>().Title, StringComparison.OrdinalIgnoreCase);
+                Assert.Contains("Orchard", contentItems.ElementAt(contentItems.Count() - 1).As<TitlePart>().Title, StringComparison.OrdinalIgnoreCase);
```

Comment:
To query the DisplayText we need to use "Content.ContentItem.DisplayText.Normalized" as the field name.
Without "Normalized" it doesn't work, this took me a while to figure out.
Is this intended?
Maybe it could be simplified to simply "DisplayText"? Would that be possible/feasible?